.github/workflows/verify.yaml

```diff
@@ -37,7 +37,8 @@ jobs:
         run: mvn test install -Dinvoker.skip -Dmaven.javadoc.skip=true -B -Dapiman.gateway-test.config=vertx3-mem
       - name: Run Tests (Vert.x 3 ES)
         run: |
-          docker run -d -p 19250:9200 -p 9300:9300 -e "discovery.type=single-node" --name=elasticsearch-7 docker.elastic.co/elasticsearch/elasticsearch-oss:7.6.2
+          echo ::set-env name=ELASTICSEARCH_VERSION::$(mvn help:evaluate -Dexpression=version.org.elasticsearch -q -DforceStdout)
+          docker run -d -p 19250:9200 -p 9300:9300 -e "discovery.type=single-node" --name=elasticsearch-7 docker.elastic.co/elasticsearch/elasticsearch-oss:${{ env.ELASTICSEARCH_VERSION }}
```

Comment:
Cool

Reply:
Would be even cooler if it would work :D
But I think we can get this also done with the new Github Actions Syntax (https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/).

Reply:
Maybe not worry about env vars at all?  How about this?

`docker run -d -p 19250:9200 -p 9300:9300 -e "discovery.type=single-node" --name=elasticsearch-7 docker.elastic.co/elasticsearch/elasticsearch-oss:`mvn help:evaluate -Dexpression=version.org.elasticsearch -q -DforceStdout``

Reply:
Works locally. :)

Reply:
For me both options are okay. It seems that @bekihm changes are also working :)